utils/docker-network-ipaddresspool.sh

```
@@ -13,27 +13,18 @@ YQ="${2:-yq}"
 numIPs=${5:-16}
+CONTAINER_ENGINE="${CONTAINER_ENGINE:-docker}"
 
 ## Parse kind network subnet
 ## Take only IPv4 subnets, exclude IPv6
 SUBNET=""
-# Try podman version of cmd first. docker alias may be used for podman, so network
-# command will be different
 set +e
-if command -v podman &>/dev/null; then
+if [[ "$CONTAINER_ENGINE" == "podman" ]]; then
   SUBNET=$(podman network inspect -f '{{range .Subnets}}{{if eq (len .Subnet.IP) 4}}{{.Subnet}}{{end}}{{end}}' $networkName)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)